docs/index.md

```diff
@@ -15,87 +15,41 @@ It offers a lot of integrations incl. Docker, Kubernetes, etc.
 
 ```bash
-kubectl create --filename manifests/
+kubectl apply --filename https://raw.githubusercontent.com/giantswarm/kubernetes-prometheus/master/manifests-all.yaml
 ```
 
-## Deploying Prometheus
-
-First, we need to create the configuration for our Prometheus. For this we use a Config Map, which we later mount into our Prometheus pod to configure it. This way we can change the configuration without having to redeploy Prometheus itself.
-
-`kubectl create --filename manifests/prometheus-configmap.yaml`
-
-Then, we create a service to be able to access Prometheus.
-
-`kubectl create --filename manifests/prometheus-services.yaml`
-
-Finally, we can deploy Prometheus itself.
+## Checking Prometheus
 
-`kubectl create --filename manifests/prometheus-deployment.yaml`
-
-Further, we need the Prometheus Node Exporter deployed to each node. For this we use a Daemon Set and a fronting service for Prometheus to be able to access the node exporters.
-
-```
-kubectl create --filename manifests/node-exporter-service.yaml
-kubectl create --filename manifests/node-exporter-daemonset.yaml
-```
-
-Wait a bit for all the pods to come up. Then Prometheus should be ready and running. We can check the Prometheus targets at https://mycluster.k8s.gigantic.io/api/v1/proxy/namespaces/default/services/prometheus/targets
+Wait a bit for all the pods to come up. Then Prometheus should be ready and running. We can check the Prometheus targets at https://<cluster-id>.k8s.gigantic.io/api/v1/proxy/namespaces/default/services/prometheus:9090/targets
 
 ![Prometheus Targets](prometheus_targets.png)
 
-## Deploying Grafana
-
-Now that we have Prometheus up and running we can deploy Grafana to have a nicer frontend for our metrics.
-
-Again, we create a service to be able to access Grafana and a deployment to manage the pods.
-
-```
-kubectl create --filename manifests/grafana-services.yaml
-kubectl create --filename manifests/grafana-deployment.yaml
-```
-
-Wait a bit for Grafana to come up. Then you can access Grafana at https://mycluster.k8s.gigantic.io/api/v1/proxy/namespaces/default/services/grafana/
-
-## Setting Up Grafana
-
-TLDR: If you don't want to go through all the manual steps below you can let the following job use the API to configure Grafana to a similar state.
-
-```bash
-kubectl create --filename manifests/grafana-import-dashboards-job.yaml
-```
-
-Once we're in Grafana we need to first configure [Prometheus](https://grafana.net/plugins/prometheus) as a data source.
+*Note*: The above URL uses your Kubernetes API to proxy to the service. As the API is guarded with your credentials, you need to [set them up in your system](/guides/accessing-services-from-the-outside/) (and/or browser). We do not recommend to set up an Ingress for Prometheus at this time, as it currently does not support any kind of authentification and thus your cluster would be open to everyone.
```

Comment:
`authentification`? I'd use `authentication`. See http://english.stackexchange.com/questions/7844/is-authentification-a-real-word